internal/sys/fs_test.go

```diff
@@ -62,12 +62,12 @@ func TestNewFSContext(t *testing.T) {
 			require.NoError(t, err)
```

Comment:
probably in this file we need to add a test that FDs are re-used. This way a small change to the file table or internals don't invalidate the overall goal which is fd re-use.

ex. 4 == open(f1) 5 == open(f2) 6 == open(f1)  close(4) 4 == open(f2)